plugins/modules/remote_management/manageiq/manageiq_policies_info.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2022, Alexei Znamensky <russoz@gmail.com>
+# Copyright (c) 2017, Daniel Korn <korndaniel1@gmail.com>
+# Copyright (c) 2017, Yaacov Zamir <yzamir@redhat.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+
+module: manageiq_policies_info
+
+short_description: Listing of resource policy_profiles in ManageIQ.
+extends_documentation_fragment:
+  - community.general.manageiq
+
+author: Alexei Znamensky (@russoz)
+description:
+  - The manageiq_policies module supports listing policy_profiles in ManageIQ.
+
+options:
+  resource_type:
+    type: str
+    description:
+      - The type of the resource to which the profile should be [un]assigned.
```

Comment:
This and the other descriptions need some update. The module does not assign or unassign.